### DISCUSSION TITLE
-m is not working when trying to pick a specific test

### DISCUSSION THREAD

**Discussion by Crocabear** [Cyfrin/foundry-full-course-cu#73] [Q&A] [answered]
<img width="342" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/129820209/e0f3f793-1777-4e25-b095-5d1b228580d7">

**Comment by alymurtazamemon** [comment, marked as answer]
@Crocabear First you run the `foundryup` again so you can get the updated version and then use the `-mt` flag because `-m` is now deprecated.
